config/gtfs-rules/SPNV-BW.no-long-distance.rule

```diff
@@ -0,0 +1 @@
+{"op":"remove", "match":{"file": "routes.txt", "route_type": "m/2|100|102|103/"}}
```

Comment:
Are there long distance trains in SPNV as well? Are they the same as those we get via DELFI eV or are they incomplete?